### PR TITLE
QA8-1287 files refactored

### PR DESCRIPTION
**Pull request by denispalkin**

### CODE REVIEW

File: test/extended/name.js [hunk 1]
```diff
@@ -16,170 +16,175 @@ describe('NAME FIELD SUITE', () => {
...
   });
 
   describe('Positive tests:', () => {
 
     it('TC-029 Name field accepts one letter', () => {
-      $(sel.name).setValue('a');
-      let alert = $(sel.error).isDisplayed();
-      expect(alert).toEqual(false);
+      $(sel.name).setValue(name.oneLetter);
+      const error = $(sel.error).isDisplayed();
```
Comment (MariiaSosedova) [inline test/extended/name.js:29]: Looks like these 2 lines use in other tests too but with different parameter of name. Maybe create a method for this or not?

File: test/extended/name.js [hunk 2]
```diff
@@ -16,170 +16,175 @@ describe('NAME FIELD SUITE', () => {
...
-      expect(alert).toEqual(false);
+      $(sel.name).setValue(name.lettersNonEnglish);
+      const error = $(sel.error).isDisplayed();
+      expect(error).toEqual(false);
     });
 
     it('TC-040 Name field input can be partially deleted', () => {
       $(sel.name).setValue(name.default);
       for (let i = 0; i < 5; i++) {
         browser.keys('Backspace'); //deletes 21804
```
Comment (MariiaSosedova) [inline test/extended/name.js:96]: for these lines we should to have method like we did with Dima

File: test/extended/name.js [hunk 3]
```diff
@@ -16,170 +16,175 @@ describe('NAME FIELD SUITE', () => {
...
+      inputValues4(name.empty, gender.it, age.default, story.comedy);
+      const submitBtn = $(sel.submit).isEnabled();
       expect(submitBtn).toEqual(false);
     });
 
-    it('TC-048 Name field input can be fully deleted', () => {
+     it('TC-048 Name field input can be fully deleted', () => {
       $(sel.name).setValue(name.default);
       for (let i = 0; i < name.default.length; i++) {
         browser.keys('Backspace');
```
Comment (MariiaSosedova) [inline test/extended/name.js:155]: method

**Review by MariiaSosedova — CHANGES_REQUESTED** [review]

File: test/extended/name.js [hunk 4]
```diff
@@ -16,170 +16,175 @@ describe('NAME FIELD SUITE', () => {
...
-      $(sel.name).setValue('                                                                       ');
-      let alert = $(sel.error).getText();
-      expect(alert).toEqual(exp.error70max);
+      $(sel.name).setValue(name.spaces71);
+      browser.pause(1000); //pause for Windows machines, not needed for Mac
+      const error = $(sel.error).getText();
+      expect(error).toEqual(exp.error70max);
     });
 
   });
```
Comment (dalaran78) [inline test/extended/name.js:190]: 190 lines in the code. No more than **150** Julia said. Could you separate Positive and Negative cases to the different files ?

**Review by dalaran78 — CHANGES_REQUESTED** [review]
Need to separate Positive and Negative cases to the different files.

File: test/extended/name.js (hunk 1 above)
Comment (denispalkin) [reply test/extended/name.js:29]: Will do

File: test/extended/name.js (hunk 2 above)
Comment (denispalkin) [reply test/extended/name.js:96]: Will do

File: test/extended/name.js (hunk 3 above)
Comment (denispalkin) [reply test/extended/name.js:155]: Will do

File: test/extended/name.js (hunk 4 above)
Comment (denispalkin) [reply test/extended/name.js:190]: Ok. Will split if the number of lines will be over 200.

**Review by MariiaSosedova — APPROVED** [review]

**Review by dalaran78 — APPROVED** [review]

**Review by MariiaSosedova — APPROVED** [review]

**Review by MariiaSosedova — APPROVED** [review]

**Review by MariiaSosedova — APPROVED** [review]

**Pull request merged**